specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing.json

```diff
@@ -3406,18 +3406,1180 @@
+            }
+          },
+          "default": {
+            "description": "Unexpected error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Transfers"
+        ],
+        "description": "Cancels a transfer request. The operation is supported only for billing accounts with agreement type Microsoft Partner Agreement.",
+        "operationId": "PartnerTransfers_Cancel",
+        "x-ms-examples": {
+          "TransferCancel": {
+            "$ref": "./examples/PartnerCancelTransfer.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/billingAccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/billingProfileNameParameter"
+          },
+          {
+            "$ref": "#/parameters/customerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/transferNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Details of canceled transfer.",
+            "schema": {
+              "$ref": "#/definitions/TransferDetails"
+            }
+          },
+          "default": {
+            "description": "Unexpected error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingProfiles/{billingProfileName}/customers/{customerName}/transfers": {
+      "get": {
+        "tags": [
+          "Transfers"
+        ],
+        "description": "Lists the transfer requests sent to a customer. The operation is supported only for billing accounts with agreement type Microsoft Partner Agreement.",
+        "operationId": "PartnerTransfers_List",
+        "x-ms-examples": {
+          "TransfersList": {
+            "$ref": "./examples/PartnerListTransfers.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/billingAccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/billingProfileNameParameter"
+          },
+          {
+            "$ref": "#/parameters/customerNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List of transfers initiated from this invoice section.",
+            "schema": {
+              "$ref": "#/definitions/TransferDetailsListResult"
+            }
+          },
+          "default": {
+            "description": "Unexpected error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/transfers/{transferName}/acceptTransfer": {
```

Comment:
The transfer suffix on accept, validate, and decline are unnecessary.  The action is being performed on a transfers resource

Reply:
I have updated as mentioned.